apps/chat/src/components/Marketplace/TabRenderer.tsx

```diff
@@ -309,10 +318,25 @@ export const TabRenderer = ({ screenState }: TabRendererProps) => {
             </>
           ) : (
             <div className="flex grow flex-col items-center justify-center">
-              <NoResultsFound iconSize={100} className="gap-5 text-lg" />
-              <span className="mt-4 text-sm">
-                {t("Sorry, we couldn't find any results for your search.")}
-              </span>
+              {selectedTab === MarketplaceTabs.MY_APPLICATIONS &&
+              !searchTerm.length ? (
```

Comment:
we should check also that there are no filters and "No models" can be teoretically for DIAL Marketplace (some error diuring models uploading)